.github/risk_level_guide.md

```diff
@@ -0,0 +1,23 @@
+## Risk level guide
+
+**Low risk**: with high confidence the change won’t unexpectedly change the behavior, or the change touches a non-critical area.
+
+**Medium risk**: with medium confidence the change won’t unexpectedly change the behavior, or the change touches an area you are not familiar with or easy to break.
```

Comment:
Do we have any (live or imaginary) examples of us merging anything other than "low risk"?

Reply:
We don't have any examples yet, and plan to work some out as the next steps. It is also expected the merge of this PR would enable engineers to use this template and generate some real-world examples :)

Reply:
I just don't think we ever merge any "risky" changes.

Reply:
Here are some examples of PR's that were medium risk+ at the time of creation:  
- #10856, numerous discussions about data races and an unexpected python test failure.
- #10595, reconciliation correctness change, modified an assertion which checks for OOO timestamps. Reverted as it introduced an unexpected test failure in test_hs32.
- #10323 adds a new generation to the code to handle a potential data corruption issue. 
- #10864, prepared update transaction Id reset issue. Reverted due to incorrectly structured conditional.